spec/Configuration/Dotenv/SchemeSpec.hs

```diff
@@ -23,13 +23,13 @@ spec = do
 
     context "when there are duplicated env configs" $
       it "should fail the check" $
-        let schemeEnvs =
+        let duplicatedEnvs =
               [ Env "FOO" (EnvType "bool") True
-              , Env "BAR" (EnvType "integer") True
               , Env "FOO" (EnvType "integer") True
               ]
-            msg = "Duplicated env variable configuration in schema: FOO"
-         in evaluate (checkScheme schemeEnvs) `shouldThrow` errorCall msg
+            schemeEnvs = Env "BAR" (EnvType "integer") True : duplicatedEnvs
+         in evaluate (checkScheme schemeEnvs)
+              `shouldThrow` (\(DuplicatedEnvs _) -> True)
```

Comment:
@CristhianMotoche I think you also need to check the content of the error message:

```hs
\(DuplicatedEnvs ["FOO"]) -> True
```